src/components/forms/common/PersonSelect.tsx

```diff
@@ -102,18 +112,29 @@ const usePersonSelect: UsePersonSelect = ({
             },
             options: personOptions,
             placeholder,
-            renderOption: (person: ZetkinPerson) => (
-                <Box alignItems="center" display="flex">
-                    <Box m={ 1 }>
-                        <Avatar
-                            src={ `/api/orgs/${orgId}/people/${person.id}/avatar` }>
-                        </Avatar>
+            renderOption: (person: ZetkinPerson) => {
+                const extraLabel = getOptionExtraLabel? getOptionExtraLabel(person) : null;
+
+                return (
+                    <Box alignItems="center" display="flex">
+                        <Box m={ 1 }>
+                            <Avatar
+                                src={ `/api/orgs/${orgId}/people/${person.id}/avatar` }>
+                            </Avatar>
+                        </Box>
+                        <Box display="flex" flexDirection="column" justifyContent="center">
+                            <Typography>
+                                { `${ person.first_name } ${ person.last_name }` }
+                            </Typography>
+                            { extraLabel && (
+                                <Typography variant="caption">
+                                    { extraLabel }
+                                </Typography>
+                            ) }
```

Comment:
We have a `ZetkinPerson` component which does exactly this! It's a tiny bit different style-wise but it also takes an optional "subtitle" or extra label etc.

Reply:
Ah, interesting! Is that the component used for task assignees? I'll try it out and see how it fits for this purpose.